Formula/docker-machine-driver-hyperkit.rb

```diff
@@ -15,6 +15,54 @@ class DockerMachineDriverHyperkit < Formula
     sha256 "92bef33ec9ad5fbdfb887fcabe550603c886065c8ec3c677732a55f84a4c7520" => :el_capitan
   end
 
+  head do
+    url "https://github.com/machine-drivers/docker-machine-driver-hyperkit.git",
+      { revision: "c72f8ba7a45adae0acf737e5e89b8e7c4710e1c8" }
```

Comment:
Head versions should not be specifying a revision

Reply:
The head version still doesn't work in macOS Catalina, and the PR had been kept open for nearly a year. I am wondering if the project had been abandoned.

This is just how I tried to make it work for me, if it cannot be merged, I don't think it's useful for adding HEAD support.